tests/process-map-input-output.nf

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env nextflow
+
+
+process foo {
+  input:
+    val id
+  output:
+    map( [id: val(id), first: path('first.txt'), second: path('second.txt')], emit: 'records' )
+  script:
+    """
+    echo 'first' > first.txt
+    echo 'second' > second.txt 
+    """
+}
+
+process bar {
+  input:
+    map( [id: val(id), first: path('first.txt'), second: path('second.txt')] )
+  output:
+    stdout
+  script:
+    """
+    echo ${id}
+    cat first.txt
```

Comment:
Would it be possible to do `cat ${first}`?

Reply:
Oh, ok, it's clear now. I wasn't sure about what `id` was `${id}` referring to, but it's not the key 😬